benchmarks/benchmark_throughput.py

```diff
@@ -191,12 +223,21 @@ def main(args: argparse.Namespace):
     parser = argparse.ArgumentParser(description="Benchmark the throughput.")
     parser.add_argument("--backend",
                         type=str,
-                        choices=["vllm", "hf"],
+                        choices=["vllm", "hf", "mii"],
                         default="vllm")
     parser.add_argument("--dataset",
                         type=str,
-                        required=True,
+                        default=None,
```

Comment:
Why does this line changed?

Reply:
Oh it's because users can set the fixed input and output lengths instead of providing a dataset.